File: py/prob_35.py
class Solution:
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        target_index = 0
        for index in range(len(nums)):
            if target > nums[index]:
                target_index += 1
        return target_index

File: cpp/prob_35.cpp
class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
        int target_index = 0;
        for (int i=0; i<nums.size(); i++) {
            if (target > nums[i]) {
                target_index++;
            };
        };
        return target_index;
    };
};